docs/build/avalanchego-apis/p-chain.md

```diff
@@ -1435,6 +1438,127 @@ curl -X POST --data '{
 }
 ```
 
+#### **JSON Example**
+
+##### **Example Call**
+
+```cpp
+curl -X POST --data '{
+    "jsonrpc": "2.0",
+    "method": "platform.getTx",
+    "params": {
+        "txID":"TAG9Ns1sa723mZy1GSoGqWipK6Mvpaj7CAswVJGM6MkVJDF9Q",
```

Comment:
Does this request/response match to each other? @joshua-kim , otherwise, can you correct the request/response to make them match?

Reply:
Has your PR with this change been deployed on Fuji? You can then try https://api.avax-test.network/